common/service/config/config.go

```diff
@@ -246,6 +255,14 @@ type (
 		RPCAddress string `yaml:"rpcAddress"`
 	}
 
+	// ReplicationConsumerConfig contains config for replication consumer
+	ReplicationConsumerConfig struct {
```

Comment:
are we going to officially support 2 means to deliver replication messages?

I guess not

Reply:
I don't think we need to support both. This is mainly for migration and testing purpose. Once we migrate to the new model we can completely remove kafka related stuffs.

Reply:
then i guess, the preference is to use dynamic config, not this.